schema/revert/warehouse/site/data.sql

```diff
@@ -1,8 +1,45 @@
--- Revert seattleflu/id3c-customizations:warehouse/site/data from pg
+-- Deploy seattleflu/id3c-customizations:warehouse/site/data to pg
 -- requires: seattleflu/schema:warehouse/site
 
 begin;
 
--- No need to revert data.
+insert into warehouse.site(identifier, details)
```

Comment:
Do we need the revert script?

Reply:
Not sure. I just left the automatically generated one from sqitch.

Reply:
Revert scripts for data modification statements (DML), as opposed to data definition statements (DDL), are tricky, particularly when they update data in place like this.

I'd either:

1. Amend the current revert script here to include statements removing the two new sites, so it's actually a revert in concept.

2. Remove the revert script entirely, acknowledging explicitly that automatic revert isn't really something we're trying to support here.

The first seems nicer.

Reply:
Deleting a site from warehouse will only work if there is encounter that references its site_id.

Reply:
I assume you meant if there is _not_ an encounter. That is true, but I think in general revert scripts for DML (not DDL) don't need to be expected to run at arbitrary points in the future, only immediately after a failed deploy (when there shouldn't be linked data yet).

Reply:
Correct, I meant _not_ an encounter.